lib/actions/account.js

```diff
@@ -0,0 +1,155 @@
+const assert = require('bsert');
+const { getClient } = require('@bpanel/bpanel-utils');
+
+const { actionsLogger: logger } = require('../helpers');
+
+const {
+  SET_WALLET_HISTORY, // TODO: misleading name...
+  SET_WALLET_ACCOUNT_INFO,
+  SET_WALLET_ACCOUNT_BALANCE,
+  SET_WALLET_ACCOUNTS,
+  USER_SELECT_ACCOUNT,
+} = require('../constants');
+
+function setAccounts(walletId, accounts) {
+  return {
+    type: SET_WALLET_ACCOUNTS,
+    payload: { accounts, walletId },
+  }
+};
+
+function setAccountHistory(walletId, accountId, history) {
+  return {
+    type: SET_WALLET_HISTORY,
+    payload: { walletId, accountId, history },
+  }
+};
+
+function setSelectAccount(accountId) {
+  return {
+    type: USER_SELECT_ACCOUNT,
+    payload: { accountId },
+  }
+};
+
+function setAccountInfo(walletId, accountId, accountInfo) {
+  return {
+    type: SET_WALLET_ACCOUNT_INFO,
+    payload: { walletId, accountId, accountInfo },
+  }
+};
+
+function setAccountBalance(walletId, accountId, balance) {
+  return {
+    type: SET_WALLET_ACCOUNT_BALANCE,
+    payload: { walletId, accountId, balance },
+  }
+};
+
+/**
+ * fetch account transaction history
+ * @param {string} - walletId
+ * @param {string} - accountId
+ * @dispatches
+ * @returns {void}
+ *
+ * TODO: add pagination
+ */
+function getAccountHistory(walletId, accountId) {
+  return async dispatch => {
+    const client = getClient();
+    logger.debug(`fetching history for ${walletId}/${accountId}`);
```

Comment:
To be honest I'm not a fan of having a lot of this type of logging on the client side. These types of things can make the browser console really cluttered imo. The actions are all pretty much already getting logged via the redux devtools anyway, which is going to be way more extensible and manageable than having loggers everywhere.

Reply:
The order of actions and the actions themselves get logged in redux devtools, but its way more flexible to have arbitrary logging. These logs are set to `debug` to prevent spam and they have been pretty useful for me while I've been developing. The normal log level is `info` and I'm not logging anything on that level. You have to opt in to see the `debug` level logs. I dislike the fact that front end development doesn't have a robust logging system, logs are not only useful to us but also any future developer/user